src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -459,18 +445,17 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
    * not adjusted.
    */
   private _scrollToOption(): void {
-    const index = this.autocomplete._keyManager.activeItemIndex || 0;
-    const labelCount = _countGroupLabelsBeforeOption(index,
-        this.autocomplete.options, this.autocomplete.optionGroups);
-
-    const newScrollPosition = _getOptionScrollPosition(
-      index + labelCount,
-      AUTOCOMPLETE_OPTION_HEIGHT,
-      this.autocomplete._getScrollTop(),
-      AUTOCOMPLETE_PANEL_HEIGHT
-    );
-
-    this.autocomplete._setScrollTop(newScrollPosition);
+    const activeItem = this.autocomplete._keyManager.activeItem;
+    if(activeItem && this.autocomplete.panel instanceof ElementRef){
+      const optionElement = activeItem._getHostElement();
+      const optionBox = optionElement.getBoundingClientRect();
+      const panelBox = this.autocomplete.panel.nativeElement.getBoundingClientRect();
```

Comment:
The reason why we're handling the scroll offset with hardcoded values was to avoid calls like these which can cause a reflow.